sdk/spring-sdk/src/it/java/com/example/wiring/SpringSdkIntegrationTest.java

```diff
@@ -490,6 +494,70 @@ public void verifyForwardHeaders() throws InterruptedException {
     Assertions.assertEquals(esHeaderValue, esResponse);
   }
 
+  @Test
+  @Ignore("waiting for the fix in sdk")
+  public void shouldNotStartTransferForWithNegativeAmount() {
+    var walletId1 = "1";
+    var walletId2 = "2";
+    createWallet(walletId1);
+    createWallet(walletId2);
+    var transfer = new Transfer(walletId1, walletId2, -10);
+
+    ResponseEntity<Void> response = webClient.put().uri("/transfer/123")
+        .bodyValue(transfer)
+        .retrieve()
+        .toBodilessEntity()
+        .block(timeout);
+
+    assertThat(response.getStatusCode()).isEqualTo(HttpStatus.BAD_REQUEST);
```

Comment:
If we are ok with using asserj then I can fix assertions in other tests, or we can we use both in parallel.